### ISSUE TITLE
no-url-protocols regex too aggressive (reports data-urls)

### ISSUE THREAD

**Issue by benthemonkey** [sasstools/sass-lint#340]
Here's the offending code (in `/lib/rules/no-url-protocols.js`):

```javascript
var isUrlRegex = /^(http(s?)):\/\/|\/\//;
```

It reports data-urls which have two consecutive slashes in them. The regex should be changed to report urls that begin with `http(s)://` or `//`. There are a number of ways to clean up the regex. Here's my suggested, simplified regex: `/^(https?:)?\/\//`
